modules/conversion-center/assets/scss/widgets/contact-buttons.scss

```diff
@@ -232,6 +242,27 @@
 		--e-contact-buttons-chat-box-bg: #FFFFFF;
 	}
 
+	&.has-platform-waze {
+		--e-contact-buttons-button-bg: #33CCFF;
+		--e-contact-buttons-button-bg-hover: #0099FF;
+		--e-contact-buttons-button-icon: #ffffff;
+		--e-contact-buttons-button-icon-hover: #ffffff;
+		--e-contact-buttons-top-bar-bg: #0099FF;
+		--e-contact-buttons-top-bar-title: #ffffff;
+		--e-contact-buttons-top-bar-subtitle: #ffffff;
+		--e-contact-buttons-close-button-color: #ffffff;
+		--e-contact-buttons-message-bubble-body: #000000;
+		--e-contact-buttons-message-bubble-time: #000000;
+		--e-contact-buttons-message-bubble-name: #000000;
+		--e-contact-buttons-message-bubble-bubble-bg: #ffffff;
+		--e-contact-buttons-message-bubble-chat-bg: #ECE5DD;
+		--e-contact-buttons-send-button-icon: #ffffff;
+		--e-contact-buttons-send-button-bg: #33CCFF;
+		--e-contact-buttons-send-button-icon-hover: #ffffff;
+		--e-contact-buttons-send-button-bg-hover: #0099FF;
+		--e-contact-buttons-chat-box-bg: #FFFFFF;
```

Comment:
Can you double check if we have system variables for some of these colours?

Reply:
I have found some of them but just for the main color, not for the hover background or text, and not for all social platforms.